automatic/git.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,25 @@
+﻿$ErrorActionPreference = 'Stop'
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+$filePath32  = "$toolsPath\Git-2.11.0-32-bit.exe"
+$filePath64  = "$toolsPath\Git-2.11.0-64-bit.exe"
+$installFile = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
+                      Write-Host "Installing 64 bit version"; $filePath64 }
+               else { Write-Host "Installing 32 bit version"; $filePath32 }
+
+$pp = Get-PackageParameters
+Set-InstallerRegistrySettings $pp
+
+Stop-GitSSHAgent
+
+$packageArgs = @{
+    PackageName    = 'git.install'
+    FileType       = 'exe'
+    SoftwareName   = 'Git version *'
+    File           = $installFile
+    SilentArgs     = "/VERYSILENT", "/SUPPRESSMSGBOXES", "/NORESTART", "/NOCANCEL", "/SP-", "/LOG", (Get-InstallComponents $pp)
+    ValidExitCodes = @(0)
+}
+Install-ChocolateyInstallPackage @packageArgs
```

Comment:
Maybe too much refactoring. You can't completely remove `Install-ChocolateyPath` - otherwise there may be instances you will get nothing on PATH (unless that has been fixed in the installer).

Not always will people pass a package parameter for the PATH set, so the old package was handling those scenarios as well (not great, but handling).